virocon/distributions.py

```diff
@@ -249,7 +461,32 @@ def _fit_lsq(self, data):
 
         
 class LogNormalDistribution(Distribution):
+    """
+    A Lognormal Distribution. 
     
+    :math:`f(x) = \\frac{1}{x\\widetilde{\\sigma} \\sqrt{2\\pi}}\\exp \\left[ \\frac{-(\\ln x - \\widetilde{\\mu})^2}{2\\widetilde{\\sigma}^2}\\right]`
+     
+    
+    Parameters
+    ----------
+    mu : float
+        Mean parameter of the lognormal distribution. Defaults to 0.
```

Comment:
I am not sure if we can state it this way. I think mu is the mean of the corresponding normal distribution, not the mean of the lognormal distribution.